log-service/src/test/kotlin/org/vivlaniv/nexohub/ApplicationKtTest.kt

```diff
@@ -15,7 +16,7 @@ fun testLoad() {
     config.useSingleServer().address = "redis://localhost:6379"
     val redissonClient = Redisson.create(config)
     for (i in 0 until 20) {
-        Thread{
+        thread {
```

Comment:
Maybe do the same thing here: make `main` `suspend`, add `coroutineScope` and do `launch` here